documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsVisibilityCell.vue

```diff
@@ -67,6 +80,20 @@ export default {
         };
       }
     },
+    isSharedWithCurrentSpace(){
+      const spaceIdentityId = eXo.env.portal.spaceIdentityId;
+      const spaceName = eXo.env.portal.spaceName;
+      const collaborators = this.file.acl.collaborators;
+      let isShared = false;
+      if (spaceIdentityId && collaborators.length > 0){
+        collaborators.forEach((element) => {
+          if (element.identity.id === spaceIdentityId &&  element.identity.name === spaceName) {
+            isShared = true ;
```

Comment:
IMO we don't need to go through the full list, we need to return directly true once the check is true